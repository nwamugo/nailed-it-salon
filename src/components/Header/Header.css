.Header {
  position: absolute;
  width: 100%;
  height: 93px;
  left: 0px;
  top: 0px;
}
.head {
  font-style: normal;
  font-weight: 600;
  font-size: 29px;
  line-height: 29px;
  color: #000000;
  position: absolute;
  height: 29px;
  left: 48px;
  top: 28px;
}
.menu {
  position: absolute;
  height: 14px;
  left: 660px;
  top: 36px;
  font-style: normal;
  font-weight: normal;
  font-size: 12px;
  line-height: 14px;
  /* identical to box height */
  text-align: center;

  color: #000000;
}

a {
  color: black !important;
}
a:hover {
  color: #f0a07f !important;
}
a.nav-link.active {
  color: #f0a07f !important;
}
