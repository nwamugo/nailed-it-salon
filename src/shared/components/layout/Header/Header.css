.Header {
  position: absolute;
  width: 100%;
  height: 5.8125rem;
}
.head {
  font-style: normal;
  font-weight: 600;
  font-size: 1.8125rem;
  line-height: 1.8125rem;
  color: #000000;
  position: absolute;
  height: 1.8125rem;
  left: 3rem;
  top: 1.75rem;
}
.menu {
  position: absolute;
  height: 0.875rem;
  left: 41.25rem;
  top: 2.25rem;
  font-style: normal;
  font-weight: normal;
  font-size: 0.75rem;
  line-height: 0.875rem;
  /* identical to box height */
  text-align: center;

  color: #000000;
}

a {
  color: black !important;
}
a:hover {
  color: #f0a07f !important;
}
a.nav-link.active {
  color: #f0a07f !important;
}

.u-margin-right {
  margin-right: -12rem;
}
