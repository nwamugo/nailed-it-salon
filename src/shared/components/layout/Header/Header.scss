.Header {
  position: absolute;
  width: 100%;
  height: 9.3rem;
}

.head {
  font-style: normal;
  font-weight: 600;
  font-size: 2.9rem;
  line-height: 2.9rem;
  color: #000000;
  position: absolute;
  height: 2.9rem;
  left: 4.8rem;
  top: 2.8rem;
}

.menu {
  position: absolute;
  height: 1.4rem;
  left: 66rem;
  top: 3.6rem;
  font-style: normal;
  font-weight: normal;
  font-size: 1.8rem;
  line-height: 1.4rem;
  /* identical to box height */
  text-align: center;

  color: #000000;
}

a {
  color: black !important;
}
a:hover {
  color: #f0a07f !important;
}
a.nav-link.active {
  color: #f0a07f !important;
}

.u-margin-right {
  margin-right: -19.2rem;
}

.u-margin-left {
  margin-left: 2rem;
}

// .Header .row {
//   @media (max-width: 26.75em) {
//     display: none;
//   }
// }
