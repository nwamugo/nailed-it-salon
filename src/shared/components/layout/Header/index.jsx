import React from 'react';
import './Header.scss';
import NailLogo from '../../../images/nail-logo.png';

function Header() {
  return (
    <div className="Header">
      <nav className="navbar navbar-expand-lg navbar-light">
        <a className="navbar-brand" href="/">
          <img className="nav-logo" src={NailLogo} width="200px" />
        </a>
        <button
          className="navbar-toggler"
          type="button"
          data-toggle="collapse"
          data-target="#navbarToggler"
          aria-controls="navbarToggler"
          aria-expanded="false"
          aria-label="Toggle navigation"
        >
          <span className="navbar-toggler-icon" />
        </button>

        <div className="collapse navbar-collapse" id="navbarToggler">
          <ul className="navbar-nav mr-auto">
            <li className="nav-item u-margin-left">
              <a
                className={`nav-link ${
                  window.location.pathname.indexOf('directions') > -1
                    ? ''
                    : 'active'
                }`}
                href="/"
              >
                HOME
              </a>
            </li>
            <li className="nav-item u-margin-left">
              <a
                className="nav-link"
                href={
                  window.location.pathname.indexOf('directions') > -1
                    ? '/#serving'
                    : '#serving'
                }
              >
                OUR SERVICES
              </a>
            </li>
            <li className="nav-item u-margin-left">
              <a className="nav-link" href="#">
                OUR STORY
              </a>
            </li>
            <li className="nav-item u-margin-left">
              <a
                className="nav-link"
                href="https://nailsaloninvisibles.simplespa.com/#"
              >
                MAKE AN APPOINTMENT
              </a>
            </li>
            <li className="nav-item u-margin-left">
              <a
                className={`nav-link ${
                  window.location.pathname.indexOf('directions') > -1
                    ? 'active'
                    : ''
                }`}
                href={
                  window.location.pathname.indexOf('directions') > -1
                    ? '/#contactus'
                    : '#contactus'
                }
              >
                CONTACT US
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  );
}

export default Header;
