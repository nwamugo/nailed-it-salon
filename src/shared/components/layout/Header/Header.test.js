import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('Header', () => {
  it('should not have regressed', () => {
    const component = shallow(<Header />);

    expect(component).toMatchSnapshot();
  });
});
