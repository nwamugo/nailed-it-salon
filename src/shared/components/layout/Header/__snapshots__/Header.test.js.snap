// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should not have regressed 1`] = `
<div
  className="Header"
>
  <nav
    className="navbar navbar-expand-lg navbar-light"
  >
    <a
      className="navbar-brand"
      href="/"
    >
      <img
        className="nav-logo"
        src=""
        width="200px"
      />
    </a>
    <button
      aria-controls="navbarToggler"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarToggler"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse"
      id="navbarToggler"
    >
      <ul
        className="navbar-nav mr-auto"
      >
        <li
          className="nav-item u-margin-left"
        >
          <a
            className="nav-link active"
            href="/"
          >
            HOME
          </a>
        </li>
        <li
          className="nav-item u-margin-left"
        >
          <a
            className="nav-link"
            href="#serving"
          >
            OUR SERVICES
          </a>
        </li>
        <li
          className="nav-item u-margin-left"
        >
          <a
            className="nav-link"
            href="#"
          >
            OUR STORY
          </a>
        </li>
        <li
          className="nav-item u-margin-left"
        >
          <a
            className="nav-link"
            href="https://nailsaloninvisibles.simplespa.com/#"
          >
            MAKE AN APPOINTMENT
          </a>
        </li>
        <li
          className="nav-item u-margin-left"
        >
          <a
            className="nav-link "
            href="#contactus"
          >
            CONTACT US
          </a>
        </li>
      </ul>
    </div>
  </nav>
</div>
`;
